keel-sql/keel-sql.gradle.kts

```diff
@@ -1,6 +1,25 @@
+import com.diffplug.gradle.spotless.SpotlessExtension
+
 plugins {
   `java-library`
   id("kotlin-spring")
+  id("ch.ayedo.jooqmodelator") version "3.5.0"
```

Comment:
This gradle plugin will use Liquibase / JOOQ / Docker to generate JOOQ classes from our schema